Directory.Build.props

```diff
@@ -31,6 +31,10 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
+  <PropertyGroup>
+    <DebugType>embedded</DebugType>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
```

Comment:
```suggestion
```

I did some local tests, and it appears that this property is indeed not necessary.